blueprints/app/files/package.json

```diff
@@ -99,8 +99,8 @@
     "eslint-plugin-prettier": "^5.1.3",
     "eslint-plugin-qunit": "^8.1.1",
     "loader.js": "^4.7.0",
-    "prettier": "^3.2.5",
-    "qunit": "^2.20.1",
+    "prettier": "^3.3.1",
+    "qunit": "^2.21.0",
     "qunit-dom": "^2.0.0",
```

Comment:
can we bump this to v3? qunit-dom@v3 is a non-ember-addon, and better for embroider/vite

Reply:
./dev/update-blueprint-dependencies.js explicitly excludes major bumps